src/liboslexec/shadingsys.cpp

```diff
@@ -3912,7 +3912,7 @@ ClosureRegistry::register_closure (string_view name, int id,
              * because we will be allocating the real struct inside it. */
             OSL_ASSERT_MSG(params[i].field_size <= int(alignof(ClosureComponent)),
                 "Closure %s wants alignment of %d which is larger than that of ClosureComponent",
```

Comment:
Beyond the scope of this review -- but would it be possible for these macros to use the fmt::printf under the hood?

Reply:
Yes, I think so. I believe when I first wrote this macro, it may have been prior to use of the formatted output template magic, and also I was a little hesitant because fmt::printf is a complicated implementation that made me nervous about calling in the middle of a panic bad enough to cause us to assert. But maybe it would be fine.